src/components/BookInfo/BookInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {
+  FunctionComponent,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+import { fetchUrl, renderAuthors } from "../../helpers/BookHelper";
+import { notification, Skeleton } from "antd";
+// @ts-ignore
```

Comment:
Is there any way to get rid of this?

Reply:
mislis na to sve? nisam ziher kak bi to napravil

Reply:
Ne ne, mislil sam samo na ts ignore haha

Reply:
nemrem, nemaju svoje typinge :(